pele/mindist/periodic_mindist.py

```diff
@@ -44,16 +41,12 @@ def align_fragments(self, coords1, coords2):
         coords1 = coords1.copy()
         coords2 = coords2.copy()
         
-        x1 = np.copy(coords1)
-        x2 = np.copy(coords2)
-
-        self.distbest = self.measure.get_dist(x1, x2)
-        ca1 = CoordsAdapter(coords=x1)
-        ca2 = CoordsAdapter(coords=x2)        
+        x1 = np.copy(coords1).reshape(-1,3)
+        x2 = np.copy(coords2).reshape(-1,3)          
         
-        dx = ca1.posRigid - ca2.posRigid
+        dx = x1 - x2
         dx -= np.round(dx / self.boxvec) * self.boxvec
-        ave2 = dx.sum(0)/ca1.nrigid 
+        ave2 = dx.sum(0)/(len(x1.flatten())/3) 
```

Comment:
FYI, the function `flatten()` copies the data into a new array.  Usually that's wasteful and you should prefer `x.ravel()` or `x.reshape(-1)`.

In this case if you just want the size of the flattened array you should simply use `x.size`

http://docs.scipy.org/doc/numpy/reference/generated/numpy.ndarray.flatten.html